x-pack/solutions/observability/plugins/observability/public/pages/overview/components/sections/ux/core_web_vitals/web_core_vitals_title.tsx

```diff
@@ -54,43 +52,26 @@ export function WebCoreVitalsTitle({
 
   return (
     <EuiFlexGroup gutterSize="none">
-      <EuiFlexItem>
-        <EuiTitle size="xs">
-          <h3>
-            {CORE_WEB_VITALS}
-            <EuiPopover
-              isOpen={isPopoverOpen}
-              button={
-                <EuiButtonIcon
-                  data-test-subj="o11yWebCoreVitalsTitleButton"
-                  aria-label={helpAriaLabel}
-                  onClick={() => setIsPopoverOpen(true)}
-                  color={'text'}
-                  iconType={'questionInCircle'}
-                />
-              }
-              closePopover={closePopover}
-            >
-              <div>
-                <EuiText>
-                  <FormattedMessage
-                    id="xpack.observability.ux.dashboard.webCoreVitals.help"
-                    defaultMessage="Learn more about"
-                  />{' '}
-                  <EuiLink
-                    data-test-subj="o11yWebCoreVitalsTitleLink"
-                    href="https://web.dev/vitals/"
-                    external
-                    target="_blank"
-                  >
-                    {CORE_WEB_VITALS}
-                  </EuiLink>
-                </EuiText>
-              </div>
-            </EuiPopover>
-          </h3>
-        </EuiTitle>
-      </EuiFlexItem>
+      <EuiFlexGroup gutterSize="s" direction="row" alignItems="center">
+        <EuiFlexItem grow={false}>
+          <EuiTitle size="xs">
+            <h3>{CORE_WEB_VITALS}</h3>
+          </EuiTitle>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiLink
+            style={{ fontSize: 14 }}
+            data-test-subj="o11yWebCoreVitalsTitleLink"
+            href="https://web.dev/vitals/"
+            external
+            target="_blank"
+          >
+            {i18n.translate('xpack.observability.ux.dashboard.webCoreVitals.help', {
+              defaultMessage: 'Learn more',
+            })}
+          </EuiLink>
```

Comment:
Instead of manually changing the `fontSize` you could wrap `EuiLink` with `EuiText`

```suggestion
       <EuiText size="s">
          <EuiLink
            data-test-subj="o11yWebCoreVitalsTitleLink"
            href="https://web.dev/vitals/"
            external
            target="_blank"
          >
            {i18n.translate('xpack.observability.ux.dashboard.webCoreVitals.help', {
              defaultMessage: 'Learn more',
            })}
          </EuiLink>
        </EuiText>
```